Release-Notes/Release2.md

```diff
@@ -116,7 +116,7 @@ allowing for long-living management nodes and workload clusters managed by them.
 the same name in the manager environment (manager-configuration.yml). All configuration
 parameters from environments/configuration.yml should be moved to environments/manager/configuration.yml.
 
-* In kolla-ansible the haproxy role was renamed to loadbalancer. Accordingly, loadbalancer must now be
+* In kolla-ansible the HAProxy role was renamed to load balancer. Accordingly, load balancer must now be
```

Comment:
The original changelog from kolla-ansible states:

"A new group loadbalancer is required in inventory file prior to upgrade. The loadbalancer group is a replacement for the haproxy group."

I'd like to stick with the upstream spelling.

Reply:
aspell disagree's but I don't mind it either way... there were several typos and I only made sure that the spelling was consistent in the whole document.

Reply:
I agree and in certain places I also put 'HAProxy' (which is the proper spelling for the project), in this case the role is named 'haproxy' and 'loadbalancer' (eg. refering to the technical role names).

Reply:
As a non-native speaker I heavly rely on aspell or web resources see https://www.spellchecker.net/load%20balancer